.gitmodules

```diff
@@ -1,12 +0,0 @@
-[submodule "testdata/case/javascript-nodejs-sample-0"]
```

Comment:
Same question here, intentional removal?

Reply:
yup. this is dead code that isn't used anywhere